src/core/components/model-example.jsx

```diff
@@ -61,39 +62,58 @@ export default class ModelExample extends React.Component {
 
     let isOAS3 = specSelectors.isOAS3()
 
-    return <div className="model-example">
-      <ul className="tab">
-        <li className={ "tabitem" + ( this.state.activeTab === "example" ? " active" : "") }>
-          <a className="tablinks" data-name="example" onClick={ this.activeTab }>{isExecute ? "Edit Value" : "Example Value"}</a>
-        </li>
-        { schema ? <li className={ "tabitem" + ( this.state.activeTab === "model" ? " active" : "") }>
-          <a className={ "tablinks" + ( isExecute ? " inactive" : "" )} data-name="model" onClick={ this.activeTab }>
-            {isOAS3 ? "Schema" : "Model" }
-          </a>
-        </li> : null }
-      </ul>
-      <div>
-        {
-          this.state.activeTab === "example" ? (
-            example ? example : (
+    return (
+      <div className="model-example">
+        <ul className="tab" role="tablist">
+          <li className={cx("tabitem", { "active": this.state.activeTab === "example" })} role="presentation">
+            <button
+              aria-selected={this.state.activeTab === "example"}
+              className="tablinks"
+              data-name="example"
+              onClick={ this.activeTab }
+              role="tab"
+            >
+              {isExecute ? "Edit Value" : "Example Value"}
+            </button>
+          </li>
+          { schema && (
+            <li className={cx("tabitem", { "active": ["model", "schema"].includes(this.state.activeTab) })} role="presentation">
+              <button
+                aria-selected={["model", "schema"].includes(this.state.activeTab)}
```

Comment:
What about adding `aria-controls` attribute to buttons and `aria-labelledby` to `tabpanel`s to connect them?

https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html

Reply:
I had originally included these attributes but they require ids to work properly. I would need access to additional unique variables so that each tabs section doesn't duplicate ids. If we can do that, I can add those attributes back to improve this

Reply:
Let's deal with this in https://github.com/swagger-api/swagger-ui/pull/7464/files#r691867303